api/kessel/relations/v1beta1/check.proto

```diff
@@ -36,5 +37,6 @@ message CheckResponse {
 		// e.g.  ALLOWED_CONDITIONAL = 3;
 	}
 	Allowed allowed = 1;
+	Zookie checked_at = 2;
```

Comment:
Is there any value to having `checked_at`, `deleted_at`, etc., rather than just `zookie`? Just wondering why you named them like this?

I'm wondering whether we want to give the impression that zookies are like timestamps, or to stay completely opaque as to what they are. I don't know the answer, just thinking out loud.

Reply:
This is a great question, it would not make much of a difference if it were all just `zookie`. The reasoning is due to the fact that I was following how authzed had setup their zedtoken. With `looked_up_at` for lookup responses https://buf.build/authzed/api/docs/main:authzed.api.v1#authzed.api.v1.LookupResourcesResponse and `checked_at` for https://buf.build/authzed/api/docs/main:authzed.api.v1#authzed.api.v1.CheckPermissionResponse etc. 

Considering it is just an opaque string, I think having names that sound like timestamps can be misleading. I think it would be good to just have a single name like `zookie` for requests and response fields and won't have users thinking they can grab some extra timestamp value from what it is. WDYT?

Reply:
> I'm wondering whether we want to give the impression that zookies are like timestamps, or to stay completely opaque as to what they are. I don't know the answer, just thinking out loud.

This – we should be very opaque about them IMO, because we can't guarantee they'll always be timestamps.

See my comment below on the "Zookie" name itself – I think we need a little less specific of a field name for these because across other FGA projects, the token we'd use isn't _actually_ a logical timestamp, so there is arguably a better name for this. Maybe simply "consistency_token" or "read_token" (similar to what you had originally).